Sources/Sentry/SentryDispatchQueueWrapper.m

```diff
@@ -3,11 +3,27 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+@interface
+SentryDispatchQueueWrapper ()
+
+@property (nonatomic, strong) dispatch_queue_t queue;
+
+@end
+
 @implementation SentryDispatchQueueWrapper
 
+- (instancetype)init
+{
+    self = [super init];
+    if (self) {
+        self.queue = dispatch_queue_create("sentry-dispatch", DISPATCH_QUEUE_SERIAL);
+    }
+    return self;
+}
+
 - (void)dispatchAsyncWithBlock:(void (^)(void))block
 {
-    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0ul), block);
+    dispatch_async(self.queue, block);
```

Comment:
For iOS 9 compatible autorelease, you could do this:

```suggestion
    dispatch_async(self.queue, ^{
        @autoreleasepool {
            block();
        }
    });
```

Reply:
Oh that makes sense. All it does is wrapping a block on `autoreleasepool` to have a more deterministic drop so this should address both points.